extensions/compressx/package.json

```diff
@@ -11,22 +11,29 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "compressVideo",
```

Comment:
Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

Reply:
Reversed. Thanks for the recommendation!